docs/README.md

```diff
@@ -8,6 +8,26 @@ layout: home
 
 Welcome to the documentation of the **Cosmos Hub application: `gaia`**.
 
+## Join Mainnet
```

Comment:
Please move these instructions into their own doc and link to it from this readme. There can be a new folder called networks with 'mainnet' and 'testnet' subfolders, and these instructions can be placed there. As well, if you want, you can create this actual script and place it in the networks folder of the root, alongside the other scripts.

Reply:
Alternatively, I'm happy to do these changes if you want.

Reply:
I can do it, **but** I think we need a quick start front and center. When I review project code, the very first thing I always want to do is run a node and indeed I judge a bit based on how easy it is. 

An alternative that I considered is a curl | bash type script or a docker image, which would leave the Readme "landing page" less busy but equally functional.

What do you think?

Reply:
I really like the curl idea! And bonus points for a docker image for connecting to mainnet (can be a separate PR if you wish).

Reply:
OK, sounds great then!